<launch>

<group ns="inmoov/" >
<rosparam file="$(find inmoov_bringup)/config/config.yaml"/>
</group>


<node pkg="inmoov_bringup" type="joint_command_dispatcher.py" name="joint_command_dispatcher" respawn="true"/>

<node pkg="inmoov_bringup" type="joint_status_dispatcher.py" name="joint_status_dispatcher" respawn="true"/>

<node pkg="inmoov_bringup" type="motor_status_dispatcher.py" name="motor_status_dispatcher" respawn="true"/>

<node pkg="inmoov_bringup" type="joint_FollowTrajectory_dispatcher.py" name="joint_dispatcher" respawn="true"/>

<node pkg="inmoov_neopixel_control" type="neopixel_command.py" name="neo_cmd" respawn="true"/>

<include file="$(find inmoov_moveit_config)/launch/demo.launch" />


<node pkg="inmoov_bringup" type="enable_manager.py" name="enable_manager" respawn="false"/>



<node pkg="inmoov_bringup" type="rviz_manager.py" name="rviz_manager" respawn="true" output="screen"/>

</launch>
